### ISSUE TITLE
Confisung behavior of S1481

### ISSUE THREAD

**Issue by hankovich** [SonarSource/sonar-dotnet#4492]
### Description

```csharp
async Task FooAsync()
{
    using var _ = await AcquireLockAsync(); // <- S1481 is reported here

    // do stuff under lock
}
```

### Expected behavior

The '_' variable is not actualy unused and I can't get rid of it (yeah, I can do `using(await AcquireLockAsync()){ /**/}`, but it looks ugly IMO). So I think this particular case is a FP.

### Related information

* SonarAnalyzer.CSharp 8.2.0-RC1

**Comment by pavel-mikula-sonarsource** [comment]
This issue was fixed a long time ago in 8.13. Please use the latest version of the analyzer 8.23 instead.
